src/Smalot/PdfParser/Font.php

```diff
@@ -122,22 +110,14 @@ public function translateChar($char, $use_default = true)
         return $use_default ? self::MISSING : $fallbackDecoded;
     }
 
-    /**
-     * @param int $code
-     *
-     * @return string
-     */
-    public static function uchr($code)
+    public static function uchr(int $code): string
```

Comment:
This change seems to have caused a problem of when the variable `$code` is not an int but rather a float.

```bash
 Smalot\PdfParser\Font::uchr(): Argument #1 ($code) must be of type int, float given, called in /home/jessegreathouse/dcol/src/vendor/smalot/pdfparser/src/Smalot/PdfParser/Font.php on line 223

  at vendor/smalot/pdfparser/src/Smalot/PdfParser/Font.php:138
    134▕
    135▕     /**
    136▕      * Convert unicode character code to "utf-8" encoded string.
    137▕      */
  ➜ 138▕     public static function uchr(int $code): string
    139▕     {
    140▕         if (!isset(self::$uchrCache[$code])) {
    141▕             // html_entity_decode() will not work with UTF-16 or UTF-32 char entities,
    142▕             // therefore, we use mb_convert_encoding() instead
```

This was most likely an unforeseen consequence of an innocent change.

Here is the pdf document for your convenience:
[15-1039-1.pdf](https://github.com/smalot/pdfparser/files/12210039/15-1039-1.pdf)

```php
<?php
use Smalot\PdfParser\Parser;
$parser = new Parser();
$pdf = $this->parser->parseFile('15-1039-1.pdf');
?>
```